bedrock/mozorg/templates/mozorg/contribute/contribute-old.html

```diff
@@ -159,9 +159,15 @@ <h2 class="tagline">{{ _('The success of our mission depends on participation fr
         <img src="{{ static('img/contribute/logos/web.png') }}" alt="">
         <h3>{{_('Web Development') }}</h3>
         <p>
+      {% if l10n_has_tag('fxos_removal') %}
```

Comment:
Let's leave this file alone. It's a 2 years old page at this point, and I think we're going to fix the contribute page in the next weeks, so I'll likely drop it completely for the few locales that never caught up with the new template.